packages/oauth/__tests__/tokenSession.test.ts

```diff
@@ -0,0 +1,103 @@
+import { describe, it, expect, vi, beforeEach } from "vitest"
+import tokenSession, { composeAuthData } from "../src/tokenSession"
+
+// Mock the functions decodeIDToken and parseIdTokenData
+vi.mock("../src/tokenHelpers", async () => {
```

Comment:
we control the function, why do we need to mock it?

Reply:
There is already tests for those functions. They are out of the scope of the session. That's why they are mocked.

Reply:
I revisit this question and I wouldn't like to now for example with default parameters should I expect from the parser function and introduce mock objects that match this defaults. I just want to concentrate on testing the `tokenSession` and `composeAuthData` without testing helpers which are already tested.

Reply:
I am stil strongly against it, it is a sign of trying to solve a coupling problem here and avoiding it, creating a future problems... solution of it is a DI pattern, please check it more out

Reply:
I think you could just call this methods :)